app/controllers/concerns/brexit_landing_page_testable.rb

```diff
@@ -0,0 +1,27 @@
+module BrexitLandingPageTestable
+  CUSTOM_DIMENSION = 68
```

Comment:
Does this have to be a number? if so perhaps we could have a comment above it about why the analytics team have given it this number or at least a clue about it. There's a little bit of a `MEANING_OF_LIFE = 42` vibe to it - that I appreciate is not confusion you've introduced but we could try alleviate for others.